### DISCUSSION TITLE
open_zarr really loads the data?

### DISCUSSION THREAD

**Discussion by aliaksei-chareshneu** [pydata/xarray#6343] [Q&A]
Dear all,

Could you tell me if open_zarr really loads the data as mentioned in the docs? Is it possible to get a data view (lazily load data) with xarray (similar to dask.from_zarr)?
https://xarray.pydata.org/en/stable/generated/xarray.open_zarr.html

Best regards,
Aliaksei

**Comment by FRidh** [comment]
`open_zarr` by default uses `dask` and thereby loads the data lazily. If you for example look at a data variable in the notebook, you can easily see you deal with chunked data. To really load in the data you can call the `.compute()` method.
